src/pages/Main/Layout/TabList/TabList.scss

```
@@ -0,0 +1,32 @@
+[class^='selectBtn'] {
+  width: calc(100% / 3);
+  display: flex;
+  align-items: center;
+  background-color: gray;
+  border-radius: 10px 10px 0 0;
+  padding: 13px;
+  color: #fff;
+
+  &.active {
+    background-color: #fff;
+    color: #000;
+
+    &.blue {
+      box-shadow: #329be7 0 4px inset;
+    }
+
+    &.green {
+      box-shadow: #a3e743 0 4px inset;
+    }
+
+    &.red {
+      box-shadow: #ff7d3a 0 4px inset;
+    }
+  }
+
+  & img {
+    margin-right: 5px;
+    width: 18px;
+    height: 18px;
+  }
```

부모 선택자(&) 사용하지 않으셔도 됩니다.
```suggestion
  img {
    margin-right: 5px;
    width: 18px;
    height: 18px;
  }
```